tst/bootstrap.php

```diff
@@ -7,7 +10,7 @@
 if (!defined('CONF')) define('CONF', PATH . 'cfg' . DIRECTORY_SEPARATOR . 'conf.ini');
 if (!is_file(CONF)) copy(CONF . '.sample', CONF);
 
-require PATH . 'lib/auto.php';
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
Is not PATH defined somewhere. Why not still use it to get an absolute path instead of a relative one?

So should not `PATH . 'vendor/autoload.php';` work?